axum-extra/src/extract/with_rejection.rs

```diff
@@ -89,7 +89,7 @@ impl<E, R> Copy for WithRejection<E, R> where E: Copy {}
 
 impl<E: Default, R> Default for WithRejection<E, R> {
     fn default() -> Self {
-        Self(Default::default(), Default::default())
+        Self(E::default(), PhantomData)
```

Comment:
I'd rather just keep `Default::default()` here and for the rest of these changes. I don't think it impacts maintainability much.

Reply:
fixed